presto-orc/src/main/java/com/facebook/presto/orc/Stripe.java

```diff
@@ -14,6 +14,7 @@
 package com.facebook.presto.orc;
 
 import com.facebook.presto.orc.metadata.ColumnEncoding;
+import com.facebook.presto.orc.reader.IntegerDictionaryProvider;
```

Comment:
Presto Orc Reader and Writer only has LongColumnWriter and LongColumnReader. This should be called LongDictionaryProvider.

Reply:
Is this an argument of semantics or convention? I think that the method is semantically integer dictionary type, it just so happens that everything upcasted to long. I don't have a strong preference here though. WDYT?